packages/earth-map/src/index.tsx

```diff
@@ -10,6 +10,9 @@ import { onRedirectCallback, onSuccessHook, onFailureHook } from './auth/hooks';
 
 import 'styles/index.scss';
 
+import 'react-app-polyfill/ie11';
+import 'react-app-polyfill/stable';
```

Comment:
This should probably be on Admin too. I know it's not react app there, but it "should" 🤞  work